services/core-web/src/components/mine/Reports/ReportPage.tsx

```diff
@@ -29,9 +29,10 @@ import { IMineReportSubmission } from "@mds/common/interfaces/reports";
 import { FORM } from "@mds/common/constants/forms";
 import { MINE_REPORT_STATUS_HASH } from "@mds/common/constants/strings";
 import { MineReportTypeUrlParam } from "@mds/common/constants/enums";
+import { useAppDispatch } from "@mds/common/redux/rootState";
 
 const ReportPage: FC = () => {
-  const dispatch = useDispatch();
+  const dispatch = useAppDispatch();
```

Comment:
wait, I see that we added an import for useAppdispatch when there was already an import for useDispatch, but that one wasn't removed. Can we take the other one out?